tests/try_scraper/test_invalid_changeset.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import mozlog
+import sys
+import unittest
+import urllib
+
+from mozdownload import TryScraper
+from mozdownload import NotFoundError
+from mozdownload.utils import urljoin
+
+import mozhttpd_base_test as mhttpd
+
+
+tests_with_invalid_changeset = [
+    # -a firefox -p linux --changeset 97322d2d2d6c
+    {'args': {'application': 'firefox',
+              'changeset': '97322d2d2d6c',
+              'platform': 'win32'}
+    },
```

Comment:
Any particular reason why you do not compare `target_url` and `target` to check that you are downloading the correct file?